XenAdmin/Wizards/CrossPoolMigrateWizard/CrossPoolMigrateNetworkingPage.cs

```diff
@@ -96,6 +96,18 @@ public override NetworkResourceContainer NetworkData(string sysId)
             return new CrossPoolMigrationNetworkResourceContainer(vifs);
         }
 
+        protected override bool AllowSriovNetwork(XenAPI.Network network, string sysId)
+        {
+            VM vm = Connection.Resolve(new XenRef<VM>(sysId));
+            if (vm == null)
+                return false;
+
+            if (vm.IsRunning())

```

Comment:
How about a VM that is suspended? Probably needs to be treated the same way are running VMs